reo/tests/test_default_outage.py

```diff
@@ -0,0 +1,299 @@
+                  "Storage": {
+                    "min_kw": 0.0,
+                    "max_kw": 0.0,
+                    "min_kwh": 0.0,
+                    "max_kwh": 1000000.0,
+                    "internal_efficiency_pct": 0.975,
+                    "inverter_efficiency_pct": 0.96,
+                    "rectifier_efficiency_pct": 0.96,
+                    "soc_min_pct": 0.2,
+                    "soc_init_pct": 0.5,
+                    "canGridCharge": True,
+                    "installed_cost_us_dollars_per_kw": 840.0,
+                    "installed_cost_us_dollars_per_kwh": 420.0,
+                    "replace_cost_us_dollars_per_kw": 410.0,
+                    "replace_cost_us_dollars_per_kwh": 200.0,
+                    "inverter_replacement_year": 10,
+                    "battery_replacement_year": 10,
+                    "macrs_option_years": 7,
+                    "macrs_bonus_pct": 1.0,
+                    "macrs_itc_reduction": 0.5,
+                    "total_itc_pct": 0.0,
+                    "total_rebate_us_dollars_per_kw": 0
+                  },
+                  "Generator": {
+                    "existing_kw": 0.0,
+                    "min_kw": 0.0,
+                    "max_kw": 0.0,
+                    "installed_cost_us_dollars_per_kw": 500.0,
+                    "om_cost_us_dollars_per_kw": 10.0,
+                    "om_cost_us_dollars_per_kwh": 0.0,
+                    "diesel_fuel_cost_us_dollars_per_gallon": 3.0,
+                    "fuel_slope_gal_per_kwh": 0.076,
+                    "fuel_intercept_gal_per_hr": 0.0,
+                    "fuel_avail_gal": 660.0,
+                    "min_turn_down_pct": 0.0,
+                    "generator_only_runs_during_grid_outage": True,
+                    "generator_sells_energy_back_to_grid": False,
+                    "macrs_option_years": 0,
+                    "macrs_bonus_pct": 1.0,
+                    "macrs_itc_reduction": 0.0,
+                    "federal_itc_pct": 0.0,
+                    "state_ibi_pct": 0.0,
+                    "state_ibi_max_us_dollars": 0.0,
+                    "utility_ibi_pct": 0.0,
+                    "utility_ibi_max_us_dollars": 0.0,
+                    "federal_rebate_us_dollars_per_kw": 0.0,
+                    "state_rebate_us_dollars_per_kw": 0.0,
+                    "state_rebate_max_us_dollars": 0.0,
+                    "utility_rebate_us_dollars_per_kw": 0.0,
+                    "utility_rebate_max_us_dollars": 0.0,
+                    "pbi_us_dollars_per_kwh": 0.0,
+                    "pbi_max_us_dollars": 0.0,
+                    "pbi_years": 0.0,
+                    "pbi_system_max_kw": 0.0
+                  }
+                },
+                "timeout_seconds": 295,
+                "user_uuid": None,
+                "description": "",
+                "time_steps_per_hour": 1,
+                "webtool_uuid": None
+              }
+            }
+
+    def test_default_outage(self):
```

Comment:
In this test it would be good to test some specific result values that relate to the utility you look up the outage duration for. Something like:

self.assertEqual(response['inputs']['Scenario']['Site']['LoadProfile']['outage_end_time_step'], response['inputs']['Scenario']['Site']['LoadProfile']['outage_start_time_step'] + x)

where x is the avg outage hr value